modules/security-jwt/src/main/java/com/devonfw/module/security/jwt/common/base/JwtAuthenticationFilter.java

```diff
@@ -21,6 +24,8 @@
  *
  * @since 2020.04.001
  */
+@Component
+@Order(SecurityProperties.BASIC_AUTH_ORDER - 1)
```

Comment:
Nice ;-)

Reply:
Trade-off: to make spring-boot starter pluggable with convention over configuration I had to find a solution. Projects are free to depend on `devon4j-security-jwt` instead of the starter and manually configure the beans (e.g. creating the filter with `new` in WebSecurityConfigurer and add to filter chain in the desired place).

Reply:
Can we define @Component here as in JwtAutoConfiguration we have defined it as Bean
 @Bean
  public JwtAuthenticationFilter jwtAuthenticationFilter() {

    return new JwtAuthenticationFilter();
  }

Same for 
 @Bean
  public JwtConfigProperties jwtConfigProperties() {

    return new JwtConfigProperties();
  }

Getting error while running mtsj
The bean 'jwtConfigProperties', defined in class path resource [com/devonfw/module/security/jwt/common/impl/JwtAutoConfiguration.class], could not be registered. A bean with that name has already been defined in URL [jar:file:/D:/projects/devon/devon-ide-scripts-3.0.0-beta24/conf/.m2/repository/com/devonfw/java/modules/devon4j-security-jwt/2020.04.001-SNAPSHOT/devon4j-security-jwt-2020.04.001-SNAPSHOT.jar!/com/devonfw/module/security/jwt/common/impl/JwtConfigProperties.class] and overriding is disabled.

same error for JwtAuthenticationFilter